pkg/cluster/ports/ports.go

```diff
@@ -27,9 +27,6 @@ const (
 	KubeletPort = 10250
 	// KubeletReadOnlyPort exposes basic read-only services from the kubelet.
 	// May be overridden by a flag at startup.
-	// This is necessary for heapster to collect monitoring stats from the kubelet
```

Comment:
I believe this is still necessary for monitoring? What makes this comment redundant?

Reply:
heapster has long been deprecated, so I thought I'd remove these comments.Did you mean to keep the 30th line?

Reply:
Something still collects stats, does it use this port? what is this port used for now?

Reply:
I'm not sure what this port is used for now, but it is marked as deprecated in the official documentation.If you think these comments are still valid, I will close this pr.

Reply:
Do you mean keeping the original comments but adding new ones, e.g. now deprecated?

Reply:
you could do that if the *port* (not just heapster) has been deprecated.

Reply:
Thanks for the suggestion, but I'm not sure I want to do this as my intention is to remove the "redundant" comments. If I do this, instead of deleting, I will be adding less necessary comments.

Reply:
The word "redundant" implies that there is something else in the very near vicinity that says almost exactly the same thing as this comment, such that no information is lost. That is not the case from my read, this comment is not redundant. It may be inaccurate or outdated, but it is not presenting duplicated information.

Reply:
okey,it's not redundant,it's outdated.